src/types/types.d.ts

```diff
@@ -19,6 +19,12 @@ export interface Keybind {
 
 export type ActionOrKeybind = Action | Keybind;
 
+export interface OpenSpaceVersion {
+  latestOpenSpaceVersion?: { major: number; minor: number; patch: number };
+  openSpaceVersion?: { major: number; minor: number; patch: number };
```

Comment:
The naming of the type is a bit confusing, since this is the actual openspace version, but the type itself is named "OpenSpaceVersion".  Consider renaming the type

Reply:
Sounds good enough to me :)